accounts/templates/logout.html

```diff
@@ -0,0 +1,7 @@
+<a id="openstax-logout" 
+   name="openstax" 
+   href="{% url 'logout' %}" 
+   id="openstax-logout">
+   <input type="button" value= "Logout you will still be logged into the Openstax OAuth2 Provider"/>
```

Comment:
The extra info in the button was really so I could make a distinction for when I was logged out of the CMS and when I was logged out of Accounts.  But I'll simplify its info.